src/docfx/build/toc/TableOfContentsMapBuilder.cs

```diff
@@ -41,14 +43,32 @@ public void Add(Document tocFile, IEnumerable<Document> referencedDocuments, IEn
         /// Build toc map including all tocs and reversed toc mapping(document -> toc)
         /// </summary>
         /// <returns>The toc map</returns>
-        public TableOfContentsMap Build()
+        public (List<Error>, TableOfContentsMap) Build()
         {
             var documentToTocs = new Dictionary<Document, HashSet<Document>>();
+            var errors = new List<Error>();
 
             // reverse the mapping between toc and documents
             // order by toc path
             var allTocs = new List<Document>();
             var experimentalTocs = new List<Document>();
+
+            // handle conflicts
+            var tocsGroupBySiteUrl = _tocToDocuments.Keys.GroupBy(k => k.SiteUrl);
+            var conflictedTocs = new HashSet<Document>();
+            foreach (var group in tocsGroupBySiteUrl)
+            {
+                var siteUrl = group.Key;
+                if (group.Count() > 1)
```

Comment:
Does this need consider version?

Reply:
@928PJY is working on TOC with versioning, maybe it can be handled in that PR.